pack/so.json

```diff
@@ -73,6 +159,60 @@
         "uniqueness": false
     },
     {
+        "code": "12030",
+        "cost": 6,
+        "deck_limit": 3,
+        "faction_code": "haas-bioroid",
+        "faction_cost": 3,
+        "flavor": "\"Submit.\"",
+        "illustrator": "Ethan Patrick Harris",
+        "keywords": "Code Gate - Bioroid - AP",
+        "pack_code": "so",
+        "position": 30,
+        "quantity": 3,
+        "side_code": "corp",
+        "strength": 4,
+        "text": "The runner can spend [click][click] to break up to 2 subroutines on Nerine 2.0.\n[subroutine] Deal 1 brain damage. The Corp may draw 1 card.\n[subroutine] Deal 1 brain damage. The Corp may draw 1 card.",
```

Comment:
Here I left it as per the card's wording. It's the recurring inconsistency in the card templates FFG is using :/